### PR TITLE
[Submission] Schulze-Forster, Doire, Richard, Badeau

### PR DESCRIPTION
**Pull request by schufo**
**Kilian Schulze-Forster**

Dear mdx workshop committee,

I request a review for the following MDX submission:

### **Unsupervised Audio Source Separation Using Differentiable Parametric Source Models:**  

**Author(s): Kilian Schulze-Forster, Clément S. J. Doire, Gaël Richard, Roland Badeau**  

**Category ***

* [ ] `POSTERS` _(+Abstract: 250 Words+one page pdf)_
  > As a participant of the MDX challenge, please know that your poster would be automatically accepted after a minimal prescreening. Hence, don't forget to provide us with the link to your team in the AIcrowd website.
  > You are also welcome to submit us some unrelated abstract, in case your interest would be in presenting some method or idea you would just like to discuss with the participants passing by.  
  > => You *don't* need to include your poster in the PR. Just be ready to have it for D-day and show it to the audience through screen sharing.
* [x] `LONG TALK`
  > A long talk will last about 20-30min, including questions. This can be about some recent paper you published, or may also present some original research and investigations you did. If you pick this category, please provide us with an extended abstract of the talk, or directly with the slides if you have them available, even in an unfinished state, at your convenience. 
* [ ] `DISCUSSIONS`
  > This category is intended to stimulate fruitful discussions and collaborations. If you pick it, it means you are willing to chair some open discussion with the participants, mainly around one particular aspect of music demixing research. For that purpose, you should provide a minimal abstract as well as a list of a few questions you would like the participants to discuss about. Don't hesitate to propose ideas, We will possibly contact you in case we identify several similar topics that were proposed!

**Preferred Timezone***

* [x] A 11:00 PM (PDT) to 9:00 AM PDT
* [ ] B 11:00 AM (PDT) to 9:00 PM PDT

  > The preferred timezone will help us to organize the day schedule. You should select at least one, but can perfectly select both if you don't care. We will organize the programme so that to maximally fit your preferences.
---

**ORGANISATION COMMITTEE (do not fill out)**

* [x] Editor acknowledgment
* [x] Reviewer 1 
* [ ] Reviewer 2
* [x] Review 1 decision [accept]
* [ ] Review 2 decision [accept/reject]
* [ ] Editor decision [accept/reject]

### CODE REVIEW

**Comment by aliutkus** [comment]
Thanks a lot for the submission

is there somewhere we could see some extended version of the content you intend to present ?

**Comment by schufo** [comment]
The talk will be about a paper which is currently in peer review. I will contact you directly to give you a private preview as I cannot share it publicly yet.

**Comment by aliutkus** [comment]
ok, great. I acknowledge good reception on this

**Review by aliutkus — COMMENTED** [review]
The original paper is still confidential and under review but I could check it and acknowledge it is of great quality. 
This work should be inspiring to the audience as a good example of few shot learning for source separation

File: paper.md [hunk 1]
```diff
@@ -1,139 +1,27 @@
...
...corresponding isolated source signals. Such datasets can be extremely costly to obtain for musical mixtures. This raises a need for unsupervised methods. We propose a novel unsupervised model-based deep learning approach to musical source separation. Each source is modelled with a differentiable parametric source-filter model. A neural network is trained to reconstruct the observed mixture as a sum of the individual source signals by estimating the source models' parameters given their fundamental frequencies. At test time, soft masks are obtained from the source signals synthesized by the source models. The experimental evaluation on a vocal ensemble separation task shows that the proposed method outperforms learning-free methods based on nonnegative matrix factorization and a supervised deep learning baseline. Integrating domain knowledge in the form of source models into a data-driven method leads to high data efficiency: the proposed approach achieves good separation quality even when trained on less than three minutes of audio. This work makes powerful deep learning based separation usable in domains where training data with isolated source signals is expensive or nonexistent.
```
Comment (aliutkus) [inline paper.md:27]: Please include a reference to the paper in submission.

**Review by aliutkus — COMMENTED** [review]
Picking accept. Please update submission asap. I think it's safe to include more details if you want: I'm pretty sure IEEE won't mind at all...

**Comment by aliutkus** [comment]
To all participants:
please note that you should update your submission by November 6th 23:00 UTC

Thank you very much

**Comment by schufo** [comment]
Thanks a lot for your feedback! I added a reference to the underlying paper in review.

**Comment by faroit** [comment]
👏 The submission has been published on midspace (the ISMIR conference platform). The submissions will also be hosted on the workshop website tomorrow. 

The schedule can be seen there or [here](https://docs.google.com/spreadsheets/d/1PaksFcHWUfNPMg-XloA3CJtrvb9uMU0erCFVyvFi9B4/edit?usp=sharing). Please let us know if there is anything missing. Otherwise we see each other on Friday/Saturday!

**Comment by faroit** [comment]
Hi @schufo, sorry for the long waiting time but I wasn't able to continue working on the proceedings before. 

I managed to finally work on cutting the video recordings of the workshop and plan to upload the recordings to YouTube very soon. In order to allow me to do this, please acknowledge the following:  

> I hereby authorize the right and permission to copyright and/or publish, reproduce or otherwise use my name, voice, and audio-visual recordings. I acknowledge and understand these materials about or of me may be used for both commercial and/or non-commercial purposes.

> I understand that my image may be edited, copied, exhibited, published and/or distributed. There is no time limit on the validity of this release nor are there any geographic limitations on where these materials may be distributed.

> I authorize that the video will be made available under the [CC BY-SA 4.0 license](https://creativecommons.org/licenses/by-sa/4.0/) on the conference website.

and comment on this by replying with `acknowledge`.

**Comment by schufo** [comment]
acknowledge

**Pull request closed without merging**